src/integration/pages/idxPage/crossPlatformTests.js

```diff
@@ -0,0 +1,100 @@
+import { runCommonCrossPlatformTests } from '../../common';
+
+export default () => {
+  runCommonCrossPlatformTests();
+
+  it('I can see the headline', () => {
+    const h1El = document.querySelector('h1');
+
+    expect(h1El).toBeInTheDocument();
+    expect(h1El.textContent).toBeTruthy();
+    expect(h1El.textContent).toMatchSnapshot();
+  });
+
```

Comment:
Could we rewrite this (and other tests) in the following format:

```suggestion
  describe('Headline', () => {
    const h1El = document.querySelector('h1');

    it('should be in the document', () => {
      expect(h1El).toBeInTheDocument();
    };
    
    it('should contain text', () => {
      expect(h1El.textContent).toBeTruthy();
    };
    
    it('should match text', () => {
      expect(h1El.textContent).toMatchSnapshot();
      };
  });

```

Reply:
Hi @karinathomasbbc , thanks for this suggestion. I'm just curious to know why we are splitting these assertions to individual tests now?

Reply:
Actually, I have just realised, there is already a headline test which you can reuse -  https://github.com/bbc/simorgh/blob/latest/src/integration/pages/liveRadio/crossPlatformTests.js#L10

Reply:
RE why to split up the assertions - it is best practice to have a single assertion per test. We should ideally only be testing one thing at a time. If a test fails, it is much easier to diagnose & fix the problem if there is only one assertion. It also makes the snapshots read much better.

Reply:
Thanks for the feedback @karinathomasbbc 

I remember putting all the assertions under the same test when I created the Front Page integration tests, mainly replicating what Jonathan initially did, but I agree this way is better. Do you know if there are any plans to follow the same approach with the rest of the integration tests?

Reply:
Hi Denis, there are indeed plans, it's just a matter of finding the time to do them! We'll get there, and I guess it might be good if we could all try to keep an eye out for any new tests, and try to ensure they follow the same new pattern. Thanks.